content/rest/channels.textile

```diff
@@ -18,6 +18,8 @@ jump_to:
   Help with:
     - Getting started#getting-started
     - Channels#channels
+    - Subscribing to a channel#subscribing
```

Comment:
@tomczoink is this intentional?  REST lib with subscribing?  We could arguably have a note that states if you want to subscribe, use the realtime lib, useful... but this leads to nowhere I think?